go.mod

```diff
@@ -7,7 +7,7 @@ replace github.com/ipfs/go-verifcid => github.com/celestiaorg/go-verifcid v0.0.1
 require (
 	cosmossdk.io/math v1.0.0-beta.3
 	github.com/BurntSushi/toml v1.2.0
-	github.com/celestiaorg/celestia-app v0.7.0-rc-1
+	github.com/celestiaorg/celestia-app v0.7.0-rc3
```

Comment:
https://github.com/celestiaorg/celestia-app/releases/tag/v0.7.0-rc3 doesn't contain the new testnode used for swamp tests so we either want to:
1. patch rc3 with it
2. upgrade to [v0.7.0-rc5](https://github.com/celestiaorg/celestia-app/releases/tag/v0.7.0-rc5)

will opt for option 2

Reply:
Option 2 won't work as-is because there are changes on branch evan/v0.7.0-rc-1-test-node (e.g. [FillBlock](https://github.com/celestiaorg/celestia-app/blob/evan/v0.7.0-rc-1-test-node/testutil/testnode/interaction_api.go#L83)) that we need to patch onto v0.7.0-rc5 or merge onto `main` and cut a new RC

@evan-forbes

Reply:
created https://github.com/celestiaorg/celestia-app/issues/860